### ISSUE TITLE
Update UseContainerItem for DF

### ISSUE THREAD

**Issue by SabreValkyrn** [RestedXP/RXPGuides#110]
Reported on latest alpha

```
Message: Interface/AddOns/RXPGuides/HSBatching.lua:34: hooksecurefunc(): UseContainerItem is not a function
Time: Sat Oct 29 10:10:07 2022
Count: 1
Stack: Interface/AddOns/RXPGuides/HSBatching.lua:34: hooksecurefunc(): UseContainerItem is not a function
[string "=[C]"]: ?
[string "=[C]"]: in function `hooksecurefunc'
[string "@Interface/AddOns/RXPGuides/HSBatching.lua"]:34: in main chu
```

**Comment by Lardeck** [comment]
Ok i found the problem. They added C_Container in the prepatch but only two functions.
![C_Container Functions](https://i.imgur.com/ktKVYdT.png)

I will make a pull request with a more robust check.

**Comment by SabreValkyrn** [comment]
Fixed by #111
